src/main/java/com/ait/lienzo/client/core/shape/wires/handlers/impl/WiresConnectorControlImpl.java

```diff
@@ -44,17 +34,13 @@
  */
 public class WiresConnectorControlImpl implements WiresConnectorControl {
```

Comment:
This class has a lot of changes but very small test coverage.
Especially:
* `onMoveStart`
* `move`
* `reset`
* `addControlPoint` - also `m_connector#addControlPoint` looks like it never will return -1, why this check is there?